populateDB.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO `dbredb`.`project` (`Name`, `Description`) VALUES ('Pacemaker', 'Pacemaker Software System');
```

Comment:
Ons begin net met die basics, populate alles en as daar tyd oor is verbeter ons dit? :)